src/dataflow/source/kafka.rs

```diff
@@ -64,12 +67,17 @@ lazy_static! {
     .unwrap();
 }
 
+#[derive(Clone, Copy)]
+struct KafkaOffset {
```

Comment:
why are there two copies of this struct? (Another one in timestamping.rs)

Reply:
Nikhil didn't want to expose stuff from inside a source to the coord/timestamping module, and it didn't feel like it should be in expr/ or in id/. Felt cleaner to have a wrapper in two places rather than export it externally.

Reply:
`dataflow-types` seems the right place to put it.

Reply:
Given it's specific to Kafka, I'm not sure that we want to expose it anywhere, even if it is at the cost of duplication?